packages/resin-event-log/test/resin-event-log.spec.js

```diff
@@ -114,8 +116,33 @@ function createGaMock(endpoint) {
 	return aggregateMock(mocks)
 }
 
+function validateGsQuery(event) {
+	return (
+		event.site_token === GOSQUARED_ID &&
+		event.api_key === GOSQUARED_API_KEY
+	)
+}
+
+function createGsMock(options, times) {
+	times = times || 1
+
+	_.defaults(options, {
+		host: GOSQUARED_HOST,
+		method: 'POST',
+		filterQuery: validateGsQuery,
+		filterBody: function() { return true },
+		response: '1'
+	})
+	delete options.event
```

Comment:
What is `event`, how does it end up in the options (I don't see it ever passed there) and why do we delete it?